brokerapi/brokers/cloudsql/postgres-definition.go

```diff
@@ -151,7 +151,7 @@ func init() {
 			{
 				FieldName: "max_disk_size",
 				Type:      broker.JsonTypeString,
-				Details:   "Maximum disk size in GB (10 minimum/default)",
+				Details:   "Maximum disk size in GB 10 is the minimum.",
```

Comment:
Should this be 2 sentences?

Reply:
Or else a well placed comma could also work :)